### PR TITLE
Update copyright information

### PR DESCRIPTION
**Pull request by JKRhb**
This PR updates the copyright information to "2013-2021" and adds a license header to two newly added files that didn't have one before.

### CODE REVIEW

**Comment by JKRhb** [comment]
(@Apollon77 Let me know if I request you too often for reviews. If this should be the case I will simply open PRs and you can have a look into them if you find the time.)

File: lib/segmentation.js [hunk 1]
```diff
@@ -1,3 +1,13 @@
+'use strict';
+
+/*
+ * Copyright (c) 2013-2021 node-coap contributors.
```
Comment (JKRhb) [inline lib/segmentation.js:4]: I was wondering if the newly added files should also have a copyright that begins in 2013? On the other hand it probably doesn't make much of a difference.

Comment (Apollon77) [reply lib/segmentation.js:4]: Formally a file copyright should start when file was added, but yes in the end ... no real dofference :-)

**Comment by Apollon77** [comment]
@JKRhb Requesting review is best  ... so I get a mail and have a better triger and see when a PR is ready ...

**Comment by JKRhb** [comment]
> @JKRhb Requesting review is best ... so I get a mail and have a better triger and see when a PR is ready ...

Okay, great :)

**Pull request merged**